addons/website/tests/test_website_visitor.py

```diff
@@ -287,40 +282,110 @@ def test_visitor_creation_on_tracked_page(self):
         # check number of visits
         self.assertEqual(visitor_portal.visit_count, 2, "There should be 2 visits for the portal user")
 
-    def test_visitor_archive(self):
-        """ Test cron archiving inactive visitors and their re-activation when
-        authenticating an user. """
-        self.env['ir.config_parameter'].sudo().set_param('website.visitor.live.days', 7)
+    def test_clean_inactive_visitors(self):
```

Comment:
Don't forget to update test methods if we update cron method name :) (thanks tde, you are really helpful)